src/endpoint/s3/s3_utils.js

```diff
@@ -79,6 +79,7 @@ const OP_NAME_TO_ACTION = Object.freeze({
     post_object: { regular: "s3:putobject" },
     post_object_uploadId: { regular: "s3:putobject" },
     post_object_uploads: { regular: "s3:putobject" },
+    post_object_select: { regular: "s3:selectobjectcontent" },
```

Comment:
Maybe I'm mixing up but is this going to affect bucket policy permissions? so is there a special bucket policy that needs to be allowed for s3 select?

Reply:
I don't see something like this.
According to https://docs.aws.amazon.com/AmazonS3/latest/userguide/selecting-content-from-objects.html you (just) need s3:GetObject.